src/parser/model/account-resource.ts

```diff
@@ -108,10 +109,16 @@ export class AccountResource extends Resource {
         }
 
         if (this.buildAccessRoleName === undefined) {
-            this.buildAccessRoleName = this.root.organizationSection.organizationRoot?.defaultBuildAccessRoleName;
+            if (this.root.developmentRole) {
+                this.buildAccessRoleName = this.root.organizationSection.organizationRoot?.defaultDevelopmentBuildAccessRoleName;
```

Comment:
I imagine that if there is a`defaultDevelopmentBuildAccessRoleName` on the root, there would be a `DevelopmentBuildAccessRoleName` on the account.

what happens if you run --dev and there is no developmentRole? maybe print a warning and default to non-dev behaviour